syntax = "proto3";

package Micro.Future.Message.Business;

message DataHeader
{
	uint32 serialId = 1;
	bool hasMore = 2;
}